clang/test/CIR/Lowering/builtin_isfpclass.c

```diff
@@ -0,0 +1,125 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.ll
```

Comment:
The separate lowering test doesn't add anything. It can be removed. We've found it more useful to have the LLVM checks in the same test with CIR checks and OGCG checks.